rolemanager/lib/Utils/globalVariables/photo_url.dart

```diff
@@ -4,4 +4,7 @@ class PhotoUrl {
 
   static String btnUrl =
       "https://cdn.discordapp.com/attachments/580125063186087966/933885364391780382/Vector_4.png";
+
+  static String invitePicure =
```

Comment:
ENG:
change variable name to "**invitePicture**"
PT:
Mudar o nome da variavel para "**invitePicture**"